params.ini

```diff
@@ -114,141 +117,319 @@ misc_flawless_topaz=0
 misc_tp_scroll=0
 misc_gold=0
 
-;=====MAGIC=====
-;helm
-magic_circlet=1
-magic_coronet=1
-magic_diadem=1
-magic_tiara=1
 
-;etc
+;==================================================================;
+;<<<<<<<<<<<<<<<<<<<<<<<<<< MAGIC ITEMS >>>>>>>>>>>>>>>>>>>>>>>>>>>;
+;==================================================================;
+
+;charms
 magic_grand_charm=1
-magic_large_charm=0
+magic_large_charm=1
 magic_small_charm=1
+
+;jewelry
 magic_jewel=1
 
-;shield
-magic_monarch=0
+;helmets
+magic_circlet=0
+magic_coronet=0
+magic_diadem=0
+magic_tiara=0
+
+;shields
+magic_monarch=1
+
+;elite paladin shields
+magic_sacred_targe=0
+magic_sacred_rondache=0
+magic_kurast_shield=0
+magic_zakarum_shield=0
+magic_vortex_shield=0
```

Comment:
Any case a magic pala shield is worth anything?

Reply:
jewelers for palas